OpenSim/Simulation/Model/Marker.h

```diff
@@ -45,7 +45,14 @@ class OSIMSIMULATION_API Marker : public Station {
     OpenSim_DECLARE_CONCRETE_OBJECT(Marker, Station);
 
 class Body;
-
+public:
+    //==============================================================================
+    // PROPERTIES
+    //==============================================================================
+    OpenSim_DECLARE_PROPERTY(fixed, bool,
+        "Flag(true or false) specifying whether or not the marker "
+        "is fixed in the marker placement step of scaling.  If false, the marker is free to move "
+        "to match its experimental counterpart.");
```

Comment:
This description makes it sound like the marker can be welded _to ground_, which I think could lead to confusion. Perhaps "Flag (true or false) indicating whether the marker is fixed in its parent Frame during the marker placement step of scaling. If false, the marker is free to move in its parent Frame to match the location of its experimental counterpart (if one exists)."?